core/src/main/java/ch/cyberduck/core/transfer/download/AbstractDownloadFilter.java

```diff
@@ -198,9 +196,9 @@ public TransferStatus prepare(final Path file, final Local local, final Transfer
                     // Free space on disk
                     long space = 0L;
                     try {
-                        space = Files.getFileStore(Paths.get(local.getParent().getAbsolute())).getUsableSpace();
+                        space = Paths.get(local.getParent().getAbsolute()).toFile().getUsableSpace();
```

Comment:
Why is this change required?

Reply:
```
Files.getFileStore("\\…") == null
```
With `Paths.get().toFile()` we get access to the FileStore of that file, thus can see usablespace for remotes.

Reply:
Do not understand your comment. It is about determining free disk space on the target volume.

Reply:
Yes, correct. If downloading to a network drive/unc path in the future `Files.getFileStore` will result in `null` being returned.

That is: This will fail to determine disk space and disable segmented downloads to `\\wsl$`, `\\wsl.localhost`, `\\?\C:\Long\Path`.